packages/flutter_tools/lib/src/commands/build_bundle.dart

```diff
@@ -28,7 +28,7 @@ class BuildBundleCommand extends BuildSubCommand {
       ..addOption('depfile', defaultsTo: defaultDepfilePath)
       ..addOption('target-platform',
         defaultsTo: 'android-arm',
-        allowed: <String>['android-arm', 'android-arm64', 'android-x86', 'android-x64', 'ios'],
+        allowed: <String>['android-arm', 'android-arm64', 'android-x86', 'android-x64', 'ios', 'darwin-x64'],
```

Comment:
Shouldn't having this in the allowed list be gated behind the desktop-enabled check?

Reply:
Done